src/main/cli/settings.h

```diff
@@ -213,7 +219,7 @@ typedef union {
 
 typedef struct clivalue_s {
     const char *name;
-    const uint8_t type;                       // see cliValueFlag_e
+    const uint16_t type;                       // see cliValueFlag_e
```

Comment:
This will increase the flash requirements for this by more than 500 bytes. I don't think this adds enough value to make this worth this much flash space usage - we should look into implementing this in a way that is not as wasteful if we really want to do it.

Reply:
@mikeller I agree, 500 bytes is a lot to sacrifice only for this.  Another (cheaper) way is to check for the osdprofile_name value name.  I don't really like that solution, but I doubt there is a better way that's not so expensive.  Happy to change it to that to move forward.

Reply:
String comparison is definitely wrong. But I think that adding OSD profiles as a new facility in settings is wrong too - there is only limited potential for other parameters to be added that would be OSD profile dependent.
We could always go the KISS way and add `osd_profile_1_name`, `osd_profile_2_name`, and `osd_profile_3_name`.

Reply:
I have to agree that implementing full OSD profile support just to get names is way overkill and also vote for the simple 3 name parameter approach. Frankly the use case (multi-quad or team/relay racing) is still a rare thing and only used by a tiny percentage of our users.

Reply:
That sounds like a good idea, a the best of a bunch of less suitable solutions, ```osd_profile_1_name, osd_profile_2_name, and osd_profile_3_name```.  I'll make the changes, let's see what we end up with.  Thanks for your input guys.

Reply:
Done.  I think this way is much better.  It also has the added benefit of backing up all the profile names on a diff/dump command instead of only the selected profile.